azerothcore/azerothcore-wotlk
# fix(Core/SpellMgr): Add cooldown to some vehicle spells

https://github.com/TrinityCore/TrinityCore/commit/4d04344b414adc965bd2760d3f30338bcce70431


## Changes Proposed:
-  Add cooldown to some vehicle spells

## Tests Performed:
- Builds
- Tested in-game



## How to Test the Changes:

Easiest way to check this is:
1. Be ally
2. .npc add temp 28468
3. mount the vehicle and use "Stampy's Stompy-Stomp" (51752) and see that you will get a cooldown

You can also test it with a quest that uses more vehicle spells:
1. .q add 10672
2. .go gob 35345
3. Use the gob and then use the spells from Arcano-Scorpo


## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
